app/app.js

```diff
@@ -103,7 +103,10 @@ if (module.hot) {
 
 // Chunked polyfill for browsers without Intl support
 if (!window.Intl) {
-  System.import('intl').then(() => render(translationMessages));
+  Promise.all([
+    System.import('intl'),
+    System.import('intl/locale-data/jsonp/en.js'),
```

Comment:
It should but only when the website support other language.
So i guess i should add new line when user run `npm run generate` new langauge then.